TiSparkR/DESCRIPTION

```diff
@@ -0,0 +1,10 @@
+Package: TiSparkR
+Type: Package
+Title: TiSpark for R
+Version: 0.1.0
+Author: Novemser
+Maintainer: The package maintainer <novemser@gmail.com>
+Description: Add a shabby thin layer to support TiSpark in R language.
+License: Apache 2.0
+Encoding: UTF-8
+LazyData: true
```

Comment:
what is lazydata?

Reply:
If the `DESCRIPTION` contains `LazyData: true`, then datasets in R will be lazily loaded. This means that they won’t occupy any memory until we use them. It is recommended that we always include `LazyData: true` in the DESCRIPTION.